sdk/servicebus/service-bus/package.json

```diff
@@ -78,6 +78,7 @@
     "@azure/core-amqp": "1.0.0-preview.3",
     "@azure/core-http": "^1.0.0",
     "@azure/ms-rest-nodeauth": "^0.9.2",
+    "@azure/identity": "1.0.0-preview.3",
```

Comment:
GA version of Identity was released back in November. Please update to use `^1.0.0` here instead of `1.0.0-preview.3`

Reply:
Was the base branch updated manually again? 
Because the one I worked with didn't surface any install step issues. And I recall checking the version of @azure/identity to use

Also, the errors were showing up after the test code was added. Did you update the version to ^1.0.0 for code in current PR or a different setup?
I will take a look at this again to use ^1.0.0 version, verify tests on local and update the PR